providers/tests/openlineage/utils/test_utils.py

```diff
@@ -43,14 +43,11 @@
 from airflow.utils.task_group import TaskGroup
 from airflow.utils.types import DagRunType
 
-from tests_common.test_utils.compat import AIRFLOW_V_2_10_PLUS, BashOperator, PythonOperator
+from tests_common.test_utils.compat import BashOperator, PythonOperator
 from tests_common.test_utils.mock_operators import MockOperator
 
 BASH_OPERATOR_PATH = "airflow.providers.standard.operators.bash"
 PYTHON_OPERATOR_PATH = "airflow.providers.standard.operators.python"
-if not AIRFLOW_V_2_10_PLUS:
```

Comment:
We don't need it any more - because when we run the tests using new "standard" provider - we will always get base operator paths from those standard providers, not from the "airlfow.operators" package :)

Reply:
Yes agree :)